docs/guides/ultimate-guides/torrents/ultimate-intro-to-torrents/index.md

```diff
@@ -0,0 +1,152 @@
+---
+slug: ultimate-intro-to-torrents
+title: The Ultimate Intro to Torrents
+tags: [torrent, tutorial]
+authors: TechSquidTV
+---
+
+# What is a Torrent?
+
+More than likely, you are at least somewhat familiar with torrents. They are a way to download files from the internet.
+But how do they work? What are the benefits of using torrents? How do you use them? Are they safe? Consider the following your guide to _torrents_.
+
+## The Background of BitTorrent
+
+In the late 90s [Bram Cohen](https://en.wikipedia.org/wiki/Bram_Cohen) ([twitter](https://twitter.com/bramcohen)), a software developer, was working for a company called [MojoNation](<https://en.wikipedia.org/wiki/Mnet_(peer-to-peer_network)#MojoNation>) that was trying to create an encrypted decentralized file storage system for commercial-use.
```

Comment:
[Nitpick] I highly recommending not using Wikipedia as a source of reference since it is known to have inaccurate information. I think using [Cohen's profile on Chia's website](https://www.chia.net/profiles/bram-cohen/) would be a better alternative if you want. 
```suggestion
In the late 90s, [Bram Cohen](https://en.wikipedia.org/wiki/Bram_Cohen) ([twitter](https://twitter.com/bramcohen)), a software developer, was working for a company called [MojoNation](<https://en.wikipedia.org/wiki/Mnet_(peer-to-peer_network)#MojoNation>) that was trying to create an encrypted decentralized file storage system for commercial-use.
```